#!/usr/bin/env python
PACKAGE = "pioneer_mrs"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# gen.add(name, type, level, description, [default, min, max])

gen.add("handpoint_offset", double_t, 0, "Set control point offset (meter)", 0.25, 0, 0.5)

exit(gen.generate(PACKAGE, "pioneer_server", "Pioneer"))
